### PR TITLE
<FunnelChart/>  - fixes

### PR DESCRIPTION
**Pull request by MayaHamo**
**API changes:**
use data's index instead of item itself.

previous signatures:
`differenceBadgeTooltipContent?: ({currentItem, nextItem, difference}: percentageDifferenceCbData) => string,
  onDifferenceBadgeTooltipShow?: ({currentItem: nextItem, difference}: percentageDifferenceCbData) => void`

new signatures:
`differenceBadgeTooltipContent?: ({currentIndex, difference}: percentageDifferenceCbData) => string,
  onDifferenceBadgeTooltipShow?: ({currentIndex, difference}: percentageDifferenceCbData) => void`

### ✅ Checklist

- 📚 Change is documented
  - [X] Story
  - [X] API description

- 🔬 Change is tested
  - [X] Component
  - [X] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6021

**Comment by shlomitc** [comment]
Hey @MayaHamo 
1. what's the motivation behind this change?
2. the visual tests are failing now, can you check what caused it and if it's the expected behavior?

**Comment by MayaHamo** [comment]
Hi @shlomitc, I spoke with @EgoziE about it,
when I tried to use this component, I figured out that in order to use `differenceBadgeTooltipContent` I need to compare between labels translations (only item { label, balue} is available, and we do not have "id" property to each item)

File: src/FunnelChart/test/FunnelChart.spec.js [hunk 1]
```diff
@@ -37,10 +37,12 @@ describe(FunnelChart.displayName, () => {
...
-      dataExample.map(async (item, index) => {
-        expect(await driver.getValueAt(index)).toBe(item.value);
-        expect(await driver.getLabelAt(index)).toBe(item.label);
-      });
+      expect(await driver.getValueAt(0)).toBe('1K');
+      expect(await driver.getLabelAt(0)).toBe('visits');
+      expect(await driver.getValueAt(1)).toBe('800');
+      expect(await driver.getLabelAt(1)).toBe('views');
+      expect(await driver.getValueAt(2)).toBe('400');
+      expect(await driver.getLabelAt(2)).toBe('cart');
```
Comment (EgoziE) [inline src/FunnelChart/test/FunnelChart.spec.js:45]: Why did you change this?

Comment (MayaHamo) [reply src/FunnelChart/test/FunnelChart.spec.js:45]: there were warnings when I ran this test, values should have been strings,
and the first value should be "1K" and not "1000"

Comment (EgoziE) [reply src/FunnelChart/test/FunnelChart.spec.js:45]: I see... I'd expect `getValueAt` to return 1000 for 1000 and 1234 for 1234
and not 1K for 1000 and 1K for 1234

Comment (MayaHamo) [reply src/FunnelChart/test/FunnelChart.spec.js:45]: for 1234 you'll get "1.2K"

**Review by EgoziE — APPROVED** [review]
